bank/bank.go

```diff
@@ -1,55 +1,83 @@
 package bank
 
 import (
+	"errors"
+	"regexp"
 	"strconv"
 )
 
-type response struct {
-	bankName string
-	isValid  bool
+var bankCode = map[string]string{
```

Comment:
This map is duplicated, we have all the data in `bank_code.go`, may I ask you please explain it? Thanks

Reply:
yes of course. but we have just two options map & slice. both of them has duplicate! but in map you can find key without use loop & it's faster than find with loop.